pdns/recursordist/syncres.hh

```diff
@@ -568,6 +568,7 @@ public:
   unsigned int d_timeouts;
   unsigned int d_unreachables;
   unsigned int d_totUsec;
+  const struct timeval d_fixednow;
```

Comment:
It's not clear to me what the difference between `d_fixednow` and `d_now` is? I guess we need a different name because `d_now` can be updated? If that's the case I wonder it we could not get rid of `setNow()` by using a new `SyncRes` object in the few related unit tests?

Reply:
It's not only the unittests, `lwres.cc:asyncresolve` also updates `d_now` via its `now` argument.

`syncres.cc` has `asyncResolveWrapper(... &d_now ...)`

Reply:
Understood, perhaps a comment about that would be nice? (on `d_now` declaration, perhaps)